src/Files.App/Helpers/UI/AppSystemBackdrop.cs

```diff
@@ -54,7 +54,16 @@ protected override void OnTargetDisconnected(ICompositionSupportsSystemBackdrop
 			base.OnTargetDisconnected(disconnectedTarget);
 			this.target = null!;
 			this.root = null!;
-			controller?.RemoveSystemBackdropTarget(disconnectedTarget);
+
+			try
+			{
+				controller?.RemoveSystemBackdropTarget(disconnectedTarget);
+			}
+			finally
+			{
+				// Ignore errors when the controller is already disposed
```

Comment:
Let’s make sure to dispose in case of something go wrong in another process.
```suggestion
				// Ignore errors when the controller is already disposed
				controller?.Dispose();
```